package.json

```diff
@@ -4,57 +4,66 @@
   "description": "A toolkit for conducting machine learning trials against confidential data",
   "license": "MIT",
   "scripts": {
-    "test": "mocha build/test/unit",
-    "pretest-functional": "npm run compile",
-    "test-functional": "mocha build/test/functional --config test/functional/.mocharc.yml",
+    "test": "mocha",
+    "pretest:functional": "npm run compile",
+    "test:functional": "mocha --config test/functional/.mocharc.yml",
     "check": "gts check",
-    "clean": "gts clean",
+    "clean": "gts clean && rimraf ./dist",
     "compile": "tsc -p .",
+    "dct": "node build/src/cli/dct.js",
+    "dct:dev": "npm run compile && node -r dotenv/config build/src/cli/dct.js",
```

Comment:
Just curious - is the ":dev" a convention or something that has semantics for npm?

Reply:
not particularly, it's just a delimiter I've seen before. To npm, it's an independent task and just an opaque string